.github/workflows/integration-test-cassandra-suite.yaml

```diff
@@ -0,0 +1,121 @@
+name: Integration test CassandraSuite
```

Comment:
With this new github action passing, we can disable cassandra tests from Jenkins. Inside this test function `func TestCassandraSuite(t *testing.T)`, we would to skip the test. The easiest might be to add an env var like `SKIP_CASSANDRA_TESTS` [here](https://github.com/rook/rook/blob/master/Jenkinsfile#L235) in the Jenksfile, then the test implementation can skip if that env var is set.

Reply:
Done!